lbrynet/core/utils.py

```diff
@@ -9,6 +9,8 @@
 
 import pkg_resources
 
+from twisted.internet import reactor
```

Comment:
choose a different reactor in reactortest1.py and check type(reactor) in reactortest2.py , you will see it will be the same reactor as initialized in reactortest1.py 

duplicate imports in python should do nothing but return the reference to the original import so it does not make sense that it would reconfigure reactor